src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -1115,6 +1115,9 @@ export class EditorHistoryEntry extends EditorQuickOpenEntry {
 
 			return true;
 		}
+		else if (mode === Mode.OPEN_IN_BACKGROUND) {
```

Comment:
I think we could always just `return super.run(mode, context)` here because the parent class does not handle `PREVIEW` anyway.